app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError
+        elif not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError
+        return f"Welcome to {self.name}"

```

Comment:
Read the checklist, add descriptive error messages:

![image](https://user-images.githubusercontent.com/104218550/209309183-cc0fbde8-92cc-4d58-9beb-0e3807d2e290.png)